modules/social_features/social_landing_page/src/SocialLandingPageConfigOverride.php

```diff
@@ -32,6 +51,29 @@ public function loadOverrides($names) {
         $overrides[$config_name] = ['datasource_settings' => ['entity:node' => ['bundles' => ['selected' => $bundles]]]];
       }
     }
+
+    $config_names = [
+      'core.entity_form_display.paragraph.section.default',
+    ];
+
+    foreach ($config_names as $config_name) {
+      if (in_array($config_name, $names)) {
+        // Grab current configuration and push the new values.
+        $config = $this->configFactory->getEditable($config_name);
+        // We have to add config dependencies to field storage.
+        $dependencies = $config->getOriginal('dependencies', FALSE)['config'];
+        $dependencies[] = 'field.field.paragraph.field_roles';
+        $overrides[$config_name]['dependencies']['config'] = $dependencies;
```

Comment:
Not necessary can update `core.entity_form_display.paragraph.section.default.yml`